### ISSUE TITLE
[API] authentication method?

### ISSUE THREAD

**Issue by theabhishek2511** [zklm/ssherder-issues#12]
There's not much mentioned about what auth method the API uses. I've tried basic auth, but it doesn't work, just says that auth credentials were not provided.

**Comment by zklm** [comment]
Right, forgot to inform people about that. Right now the only authentication method for the user API is session authentication, which is only suitable for making requests from the site while the user is logged in (used for adding/editing/deleting characters/formations). 

I'm still looking in to and testing basic/token authentication.
